packages/web/components/swap-tool/index.tsx

```diff
@@ -106,10 +106,14 @@ export const SwapTool: FunctionComponent<SwapToolProps> = observer(
     const routesVisDisclosure = useDisclosure();
 
     const [showQuoteDetails, setShowEstimateDetails] = useState(false);
-    /** User has input and there is enough liqudity and routes for given input. */
-    const isQuoteDetailRelevant =
+
+    const isInputAmountGreaterThanZero =
```

Comment:
FYI `inAmountInput` has an `isEmpty` flag that may be useful. Not sure though.

Reply:
I'll just revert it to the changes in prod.